### ISSUE TITLE
Migrate email services from shenron to fafnir

### ISSUE THREAD

**Issue by grischard** [openstreetmap/operations#782]
* Outgoing mail server
- [x] Wiki
- [x] Dev (faffy)
- [x] Community (Discourse)

- [x] Incoming MX

* Mailing lists server
* Mailman
* Pipermail

**Comment by tomhughes** [comment]
That's really only two things - exim and mailman.

We already have exim migrated and some outbound mail is using it. Inbound would work as well - any unknown addresses (like mailman lists) get bounced over to shenron.

Mailman I need to finish figuring out.

**Comment by tomhughes** [comment]
All incoming and outgoing mail is now going through fafnir but for now mailing list mail is being passed to shenron to be processed by mailman and then back to fafnir to go back out.

**Comment by Firefishy** [comment]
I think the mailman migration could be managed via a separate ticket.

**Comment by grischard** [comment]
Blocked by #840 and #149

**Comment by grischard** [comment]
Closing, tracking the rest in #840 (mailman migration)
